src/components/sidebar/Sidebar/styles.module.css

```
@@ -46,6 +46,8 @@
 .drawerButton {
   position: absolute !important;
   z-index: 2;
+  color: white;
```

We should support both dark and light modes:

```suggestion
  color: var(--color-text-primary);
```